### PR TITLE
Fixed Issue Number #327 (To allign inputs in matrix and also in some other section in center )

### PR DESCRIPTION
**Pull request by abhijeet141**
Closes: #[issue number that will be closed through this PR]
Issue Number #327 

#### Describe the changes you've made

I have alligned input in matrix and also in some other section

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] My changes generate no new warnings.

## Screenshots

|        Original         |          Updated           |
| :---------------------: | :------------------------: |
| **original screenshot** | <b>updated screenshot </b> |
image
image

### CODE REVIEW

File: index.html
Comment (satcasm) [inline index.html:3080]: Use relative units such as ```vh vw rem em``` instead of ```px``` to ensure page stays responsive

File: index.html
Comment (satcasm) [inline index.html:3088]: Use relative units such as ```vh vw rem em``` instead of ```px``` to ensure page stays responsive

Comment (abhijeet141) [reply index.html:3088]: Okay Thanks for your suggestion🙂

**Comment by shrey27tri01** [comment]
@abhijeet141 What's the progress on this? Anything you're stuck on?

**Comment by abhijeet141** [comment]
> @abhijeet141 What's the progress on this? Anything you're stuck on?
Actually i have made changes but it is showing that there are conflicts

**Comment by sairish2001** [comment]
@satcasm How you reviewed our PR?

**Comment by satcasm** [comment]
@sairish2001 was I not supposed to? If so my bad. ;)

**Comment by shrey27tri01** [comment]
@satcasm While we do appreciate your eagerness to help others, you must understand that reviewing the PRs of fellow participants without the approval of the repository owner or the maintainers is bad practice. Hope you understand, and please refrain from doing so in the future. But we really hope you help new contributors contribute to this repository in other ways!

**Comment by satcasm** [comment]
@shrey27tri01 thanks for the heads up and I apologize for not asking prior permission.

**Comment by shrey27tri01** [comment]
@satcasm No issues at all!

**Review by shrey27tri01 — CHANGES_REQUESTED** [review]
@abhijeet141 There are a lot of unnecessary changes in the file `appjs/matrixlogic.js`, like addition/removal of spaces. Please remove these, since they are not required.

Moreover, in the file `index.html`, you have used the `<center>` tag, which has ben deprecated (refer [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center)). Please replace the usage of `<center>` tags with alignment through css.

**Comment by abhijeet141** [comment]
> @abhijeet141 There are a lot of unnecessary changes in the file `appjs/matrixlogic.js`, like addition/removal of spaces. Please remove these, since they are not required.
> 
> Moreover, in the file `index.html`, you have used the `<center>` tag, which has ben deprecated (refer [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center)). Please replace the usage of `<center>` tags with alignment through css.

I will make all changes 🙂

**Comment by abhijeet141** [comment]
I am closing this PR and will make a new PR :)

**Pull request closed without merging**
